### ISSUE TITLE
Support pinning in composite actions

### ISSUE THREAD

**Issue by cmur2** [mheap/pin-github-action#44]
I wanted to use this awesome tool to pin Github Actions dependencies in a [composite action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action) like:
```yaml
name: Deploy app

inputs:
  target_environment:
    description: The env to deploy to
    required: true
  github_token:
    description: Credential (token) needed to push Git tags to current repository
    required: true

runs:
  using: composite
  steps:
  - name: Create GitHub deployment
    uses: bobheadxi/deployments@v0.6.1
    id: deployment
    with:
      step: start
      token: ${{ inputs.github_token }}
      env: ${{ inputs.target_environment }}

# etc
```

I did `npm install -g pin-github-action` followed by `pin-github-action .github/actions/deploy-app/action.yml` (by convention I store composite actions in `.github/actions` but the path shouldn't matter) which yields the error: `No jobs found`

I also tried running `pin-github-action` on a workflow that uses my composite action but as expected according to [how it works](https://github.com/mheap/pin-github-action#how-it-works) any step using a `./local-path` action is skipped so nothing changed.

It would be cool to have support for composite actions as they are a really nice feature of GHA and structurally not too different from workflows 🙇🏼 (I might send a PR but am not fluent in JavaScript)

**Comment by mheap** [comment]
This is now published to NPM as version `1.5.0`. Thanks for raising the issue!

**Comment by cmur2** [comment]
Thank you, I tested and it works like a charm :+1:
